hw/ip/prim/util/sparse-fsm-encode.py

```diff
@@ -45,9 +58,19 @@ def main():
                         help='Custom seed for RNG. Can be used to make '
                              'subsequent runs deterministic. The script '
                              'randomly picks a seed if not specified.')
+    parser.add_argument('--language',
+                        choices=['sv', 'c'],
+                        default='c',
```

Comment:
Seems a bit cruel to take the HW tool and change to output C by default. :P

Reply:
Can we make this SV by default? I think we're going to use it more widely for HW ;)...

Reply:
Yeah this is an oops. The intention is that it's sv by default (and you'll see i didn't add the `--language` to the sv template).